web/webpack.prod.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     path: path.resolve(__dirname, '../docs'),
     filename: 'bundle.js',
   },
+  node: {
+    fs: 'empty'
```

Comment:
this is a hack, right? I had to do this at work - maybe add a quick comment about what this is for?

Reply:
Also, instead of duplicating the config you can just grab `config.node` so that we can just change it in one place if needed

Reply:
Sounds good